csharp/WriterProperties.cs

```diff
@@ -9,6 +9,79 @@ namespace ParquetSharp
     /// </summary>
     public sealed class WriterProperties : IDisposable
     {
+        /// <summary>
+        /// Represents a column sorting specification.
+        /// </summary>
+        public readonly struct SortingColumn : IEquatable<SortingColumn>
+        {
+            /// <summary>
+            /// Creates a new sorting column specification.
+            /// </summary>
+            /// <param name="columnIndex">The index of the column to sort by</param>
+            /// <param name="isDescending">Whether to sort in descending order (true) or ascending order (false)</param>
+            /// <param name="nullsFirst">Whether nulls should come first (true) or last (false)</param>
+            public SortingColumn(int columnIndex, bool isDescending, bool nullsFirst)
```

Comment:
For convenience I think this could have some default values:
```suggestion
            public SortingColumn(int columnIndex, bool isDescending = false, bool nullsFirst = false)
```